src/GitVersionExe/GitVersionOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace GitVersion
+{
+    public class GitVersionOptions
```

Comment:
What's the plan with `GitVersionOptions`?

Reply:
I forgot to delete it. Removed